test/test_suite.cpp

```diff
@@ -1228,8 +1252,8 @@ TEST_CASE("float", "[]" ) {
 #ifndef PRINTF_DISABLE_SUPPORT_EXPONENTIAL
   // brute force exp
   str.setf(std::ios::scientific, std::ios::floatfield);
-  for (float i = -1e20; i < 1e20; i += 1e15) {
-    test::sprintf(buffer, "%.5f", i);
+  for (float i = -1e20; i < (float)1e20; i += (float)1e15) {

```

Comment:
I think instead of casting you can use the F suffix to indicate it is a float.

Reply:
Yes, thanks!  :-)